src-built-in/components/windowTitleBar/src/windowTitleBar.jsx

```diff
@@ -1,434 +1,11 @@
-		let titleWrapperClasses = "fsbl-header-center";
-		let rightWrapperClasses = "fsbl-header-right";
-		let tabRegionClasses = "fsbl-tab-area";
-		let headerClasses = "fsbl-header";
-
-		//If we're showing tabs, we throw these classes on to modify styles.
-		if (this.state.showTabs) {
-			headerClasses += " fsbl-tabs-enabled";
-		}
-		if (this.state.tabs.length > 1) {
-			headerClasses += " fsbl-tabs-multiple";
-		}
-		//See this.allowDragOnCenterRegion for more explanation.
-		return (
-			<div className={headerClasses}>
-				{/* Only render the left section if something is inside of it. The left section has a right-border that we don't want showing willy-nilly. */}
-				{RENDER_LEFT_SECTION &&
-					<div className="fsbl-header-left">
-						{self.state.showLinkerButton ? <Linker /> : null}
-						{self.state.showShareButton ? <Sharer /> : null}
-					</div>
-				}
-				{/* center section of the titlebar */}
-				<div className={titleWrapperClasses}
-					ref={this.setTabBarRef}>
-					{/* If we're supposed to show tabs and the window isn't babySized */}
-					{!this.state.showTabs && <div className='fsbl-header-center'>{this.state.windowTitle}</div>}
-					{this.state.showTabs &&
-						<TabRegion
-							onTabDropped={this.allowDragOnCenterRegion}
-							className={tabRegionClasses}
-							thisWindowsTitle={this.state.windowTitle}
-							boundingBox={this.state.tabBarBoundingBox}
-							listenForDragOver={!this.state.allowDragOnCenterRegion}
-							tabs={this.state.tabs}
-							ref="tabArea"
-							onTitleUpdated={this.resizeDragHandle}
-						/>}
-
-				</div>
-				<div className={rightWrapperClasses} ref={this.setToolbarRight}>
-					{this.state.alwaysOnTopButton && showMinimizeIcon ? <AlwaysOnTop /> : null}
-					<BringSuiteToFront />
-					{this.state.minButton && showMinimizeIcon ? <Minimize /> : null}
-					{showDockingIcon ? <DockingButton /> : null}
-					{this.state.maxButton ? <Maximize /> : null}
-					{this.state.closeButton ? <Close /> : null}
-				</div>
-			</div>
-		);
-	}
-}
-
-// This is how we used to do it, but this was causing timing problems in windows that
-// reload, such as Symphony. FSBL.addEventListener() is a better approach because
-// it is pub/sub, if the event had fired in the past then it will still be fired.
-// window.addEventListener("FSBLReady", function () {
-
-if (window.FSBL && FSBL.addEventListener) { FSBL.addEventListener("onReady", FSBLReady); } else { window.addEventListener("FSBLReady", FSBLReady) }
-function FSBLReady() {
-	if (FSBL.titleBarInserted) return;
-	FSBL.titleBarInserted = true;
-	storeExports.initialize(function () {
-		HeaderActions = storeExports.Actions;
-		windowTitleBarStore = storeExports.getStore();
-		ReactDOM.render(<WindowTitleBar />, document.getElementById("FSBLHeader"));
-	});
-}
+function init() {
+	if (window.headerLoaded) return;
+	window.headerLoaded = true;
+	require("./windowTitleBar2.jsx");

```

Comment:
rename this file to `entry.jsx` and keep the other as windowTitleBar.jsx. Then change the webpack. titlebar and titlebar2 is   gross.